crates/vizia_core/src/systems/accessibility.rs

```diff
@@ -0,0 +1,224 @@
+use std::sync::Arc;
+
+use crate::{accessibility::IntoNode, prelude::*};
+use accesskit::{
+    kurbo::Rect, Node, NodeId, TextDirection, TextPosition, TextSelection, TreeUpdate,
+};
+use cosmic_text::Edit;
+use unicode_segmentation::UnicodeSegmentation;
+use vizia_storage::LayoutTreeIterator;
+
+// Updates node properties from view properties
+// Should be run after layout so that things like bounding box are correct
+// This system doesn't change the structure of the accessibility tree as this is done when views are built/removed
+// TODO: Change this to incrementally update nodes when required instead of updating all nodes every frame
```

Comment:
I haven't profiled it, but accesskit is designed for the case where the nodes need to be updated every frame to accommodate immediate mode GUIs. However, since we do know what changes it probably wouldn't be difficult to update only the nodes that changed. It's just whether or not we want to block on that.